content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -3234,22 +3234,23 @@ Set a timestamp of the time when a record is created.
 
 - Implemented at Prisma level
 
-### <inlinecode>dbgenerated()</inlinecode>
+### <inlinecode>dbgenerated(...)</inlinecode>
 
 Represents **default values** that cannot be expressed in the Prisma schema (such as `random()`).
 
 #### Remarks
 
 ##### Relational databases
 
-- Compatible with any type
-- If a value is present, it cannot be empty (for example, `dbgenerated("")`) - [2.21.0](https://github.com/prisma/prisma/releases/tag/2.21.0) and later
+- Compatible with any scalar type
+- Needs a value, `dbgenerated()` is not allowed in [TODO](https://github.com/prisma/prisma/releases/tag/TODO) and later
```

Comment:
TODO later

Reply:
@janpio @Jolg42 do you know if we can point to a specific release here now?

Reply:
No this will only happen one day in the far future probably as we have some actual users of this.

Reply:
if this is the only thing that blocks this PR, let's just delete the bullet and merge?